packages/components/src/components/positions-drawer-card/positions-drawer-card.jsx

```diff
@@ -51,14 +53,14 @@ const PositionsDrawerCard = ({
     const contract_card_classname = classNames('dc-contract-card', {
         'dc-contract-card--green': !is_turbos && !is_multiplier && profit_loss > 0 && !result,
         'dc-contract-card--red': !is_turbos && !is_multiplier && profit_loss < 0 && !result,
+        'dc-contract-card--turbos': is_turbos && is_open_positions,
```

Comment:
@hamza-deriv this classname should be applied only for open positions but not for recent positions or contract details, right?

Reply:
yes exactly